addons/web/static/src/js/views/calendar/calendar_model.js

```diff
@@ -382,8 +383,9 @@ return AbstractModel.extend({
             allDayText: _t("All day"),
             views: {
                 week: {
-                    columnFormat: 'ddd ' + time.getLangDateFormat(),
-                    titleFormat: time.getLangTimeFormat(),
+                    // For the mobile devices the date format should be the day following by date else it should as per the user prefered format.
```

Comment:
So, we don't use the user prefered format on mobile? Could you add a small comment to explain why?

Reply:
That is because we want to display the entire calendar on a single screen if we display the user prefered date format, the columns overlap each other and for doing so we have to either decrease the font size or we can display day following by date.
see the link for ref: https://drive.google.com/open?id=1yLrXo_HsFqv28exc0VafM1dDkKtespeQ
as per the discussion in this task, we have selected the second option and to archive that we have added this line.

Reply:
That is because we want to display the entire calendar on a single screen if we display the user preferred date format, the columns overlap each other and for doing so we have to either decrease the font size or we can display day following by date.
see the link for ref: https://drive.google.com/open?id=1yLrXo_HsFqv28exc0VafM1dDkKtespeQ
as per the discussion in this task, we have selected the second option and to archive that we have added this line.

Reply:
Ok, but just add this explanation in the js comment.
You can write something like that:
"Hard coded date format to avoid columns overlapping in mobile."